dynawo/sources/Modeler/DataInterface/DYNComponentInterface.h

```diff
@@ -136,6 +136,12 @@ class ComponentInterface {
    */
   virtual bool isConnected() const;
 
+  /**
+   * @brief check the connection state of the component
+   * @return @b true is the component is connected at least at one end, @b false else
```

Comment:
typo, true if

Reply:
Fixed comment in `isConnected` and `isPartiallyConnected` methods.